operations/addresses.md

```diff
@@ -40,62 +45,59 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
+      "AccountId": "3db2c989-7d95-42b4-a502-a9f246db1634",
+      "AccountType": "Customer",
+      "Line1": "I.P. Pavlova 5",
+      "Line2": null,
+      "City": "Prague",
+      "PostalCode": "12000",
+      "CountryCode": "CZ",
+      "CountrySubdivisionCode": null,
+      "Latitude": null,
+      "Longitude": null,
+      "UpdatedUtc": "2023-11-29T14:49:29.982Z",
+      "IsActive": true
+    }
+  ]
 }
 ```
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `Addresses` | array of [Account address](#account-address) | required | The collection of Account addresses, containing address and account information. |
-| `Cursor` | string | required | Unique identifier of the last and hence oldest address item returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older Account address. |
+| `Addresses` | array of [Account address](addresses.md#account-address) | optional | The collection of Account addresses, containing address and account information. |
+| `Cursor` | string | optional | Unique identifier of the last and hence oldest address item returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older Account address. |
 
 #### Account address
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `Id` | string | required | Unique identifier of the address. |
-| `ChainId` | string | optional | Unique identifier of the chain. |
 | `AccountId` | string | required | Unique identifier of a [Company](companies.md#company) or a [Customer](customers.md#customer) within the enterprise. |
-| `AccountType` | string | required | A discriminator specifying the [type of account](accounts.md#account-type), e.g. customer or company. |
-| `Line1` | string | optional | First line of the address. |
-| `Line2` | string | optional | Second line of the address. |
-| `City` | string | optional | The city. |
-| `PostalCode` | string | optional | Postal code. |
-| `CountryCode` | string | optional | ISO 3166-1 code of the [Country](countries.md#country). |
-| `CountrySubdivisionCode` | string | optional | ISO 3166-2 code of the administrative division, e.g. `DE-BW`. |
+| `ChainId` | string | optional | Unique identifier of the chain. |
+| `AccountType` | [Account type](accounts.md#account-type) | required | A discriminator specifying the [type of account](accounts.md#account-type), e.g. customer or company. |
+| `Line1` | string | optional, max length 255 characters | First line of the address. |
+| `Line2` | string | optional, max length 255 characters | Second line of the address. |
+| `City` | string | optional, max length 255 characters | The city. |
+| `PostalCode` | string | optional, max length 255 characters | Postal code. |
+| `CountryCode` | string | optional, max length 3 characters | ISO 3166-1 code of the [Country](countries.md#country). |
```

Comment:
Are you sure about that, @VBuben? To my knowledge ISO 3166-1 codes are up to 3 characters depending on whether you use alpha-2 or alpha-3 variant.

However, I would consider dropping the MaxLength constraint altogether, the format of the field is defined by ISO standard and that can change.

Reply:
I'm no expert, but according to Wikipedia there are three variants Alpha-2, Alpha-3 and Numeric, which are 2, 3 and 3 characters respectively. We should say which one we are using. I assume it's the most common Alpha-2, because the example is "CZ".

Reply:
ah yes @jonas-paul, either link to it or better to just use the same description.